src/components/Navbar/index.tsx

```diff
@@ -20,25 +20,85 @@ const Navbar: FC = () => {
   })
 
   const navLinks: LinkInfo[] = [
```

Comment:
We can move it outside the component definition to avoid creating an array on every render.